### ISSUE TITLE
Filter on single runs page - to narrow down annotations

### ISSUE THREAD

**Issue by junxini** [chanzuckerberg/cryoet-data-portal#44]
Add the following 6 filters to the single run page to narrow down annotations:
  - 1.Annotation Author 
    - new filter following existing patterns, should return annotation authors, and not dataset authors 
  - 2.Object Name
    - filter exists on Browse All page
  - 3.GO ID
    - new filter, see filter specs in designs
  - 4.Object Shape Type
    - filter exists on Browse All page
  - 5.Method Type
    - new filter following existing patterns. Should only have a max of 3 dropdown options
    - dependent on #443 and #442  
  - 6.Annotation Software
    - new filter following existing patterns

**Link to design:** https://www.figma.com/file/q6Z394Xy6wUmaXQ9YFjZH7/Kevin---2023-Spillover-Work?type=design&node-id=2118-11276&mode=design&t=W43r2ZQSbiy3s1ES-0

**Comment by junxini** [comment]
point changed from 6 to 3 because E2E testing efforts is being captured in a separate ticket
